### ISSUE TITLE
@if does not allow string result expressions

### ISSUE THREAD

**Issue by earthshrink** [andmarti1424/sc-im#541]
Entering `@if` with string result expressions, as in this example from the help file `@if(A1>100,"over","not enough")` leads to `Illegal numeric expression` error. The help file says `Remember string expressions should be entered by typing \"`. But prefixing the quote with back-slash, as in entering `@if(1=1,\"y",\"n")` results in `syntax error`.
It does take `@if(1=1, G1, G2)`, but it pulls the number and not the string in G1 or G2. I get `0`, even though G1 contains the label `true` and no numeric value. Please advise if this works and what I am doing wrong.
thanks.

**Comment by andmarti1424** [comment]
You have to enter the whole formula with `\"`
and then:
`@if(A1>100,"over","ne")`

**Comment by earthshrink** [comment]
Ah, in place of `=`. Thanks, this works!

**Comment by earthshrink** [comment]
Unfortunately, when I quit the spreadsheet, it adds numeric value `0.00` to each such cell. So at next open, it shows `over0.00` or `ne0.00`, etc.

**Comment by earthshrink** [comment]
Workaround: `:format " "` (space required so it will be saved).

**Comment by andmarti1424** [comment]
Yep @earthshrink. That was an issue when saving the file rather than in the evaluation. Please update to latest commit on new branch and try again. Thanks.

**Comment by andmarti1424** [comment]
All stuff from new branch was merged to default freeze branch. So update to latest commit in there. Thanks.

**Comment by earthshrink** [comment]
Updated to latest commit, it works- my workaround not needed.
thanks so much.
